src/main/java/io/ipfs/multibase/BaseN.java

```diff
@@ -0,0 +1,60 @@
+package io.ipfs.multibase;
+
+import java.math.BigInteger;
+
+/**
+ * @author Simone I. Conte "sic2@st-andrews.ac.uk"
```

Comment:
I like the idea of this class, but 99% of the code is not yours, so probably better off without the explicit author comment

Reply:
oh sorry, IntelliJ put it there. I did not mean to gain authorship of this.